pages/what-to-expect.vue

```diff
@@ -25,9 +25,6 @@
         <p>We've tried this before and it's been really well received. And if it sounds a bit scary, remember you can always pass!</p>
       </div>
       <h4><b>8pm</b>: quick break</h4>
```

Comment:
Do we want to leave in the 8pm quick break? We sometimes have one if the thinking round has been very intense but normally we go straight into the next bit at 8pm.

Reply:
hmmm... good question. maybe we could say

8pm: Optional break. Sometimes we take a short break if it feels like we need one after the thinking round. Otherwise we go straight into the roundtable chat.

Reply:
Good call. Happy to make that change?

Reply:
@foleynotrose Just a reminder that this is ready for a small update.

Reply:
updated!